.travis.yml

```diff
@@ -1,25 +1,33 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
+## Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 
 os:
   - linux
+  - osx
 
 julia:
   - 1.0
+  - 1.2
+  - 1.3
+  - nightly
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - julia: nightly  # development versions
+
+# avoid duplicate tests in PRs
+branches:
+  only:
+    - master
 
 notifications:
   email: false
 
-jobs:
-  include:
-    - stage: "Documentation"
-      julia: 1.0
-      os: linux
-      script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()));
-                                               Pkg.instantiate();'
-        - julia --project=docs/ docs/make.jl
-      after_success: skip
+# https://github.com/travis-ci/travis-ci/issues/4942 workaround
+git:
+  depth: 99999999
```

Comment:
The issue got closed, so apparently this is not needed anymore?

Reply:
ok, yes i think so ... :+1: